### ISSUE TITLE
Model scaling based on latitude

### ISSUE THREAD

**Issue by joedjc** [jscastro76/threebox#202]
Hi,

Whilst this touches on a wider issue about map projection, an issue we face is that the real-world size of 3D models varies quite significantly based on latitude/distance from the equator. 

In threebox the model size is fixed. It means if we have a model of a car which is correctly scaled in one location, it can be a very different scale if it's place somewhere else in the world and therefore inaccurate.

Would it be possible to account for this within threebox automatically, rather than having to account for the model scale separately?

**Comment by jscastro76** [comment]
There is a reason for that, Mapbox Mercator projection defines what how long a meter is in different latitudes. 
If you set `scale: 'scene’` instead of `scale: 'meters’` The object will never rescale into latitude. But obviously the value will be very different to the value in meters

**Comment by joedjc** [comment]
Thanks @jscastro76. So if I understood correctly if I set the scale in meters and add a model, it should be at the correct size for that location. However, if I move that object the scale will not re-adjust? The same would apply if I duplicate an object somewhere else in the world?

**Comment by jscastro76** [comment]
sorry I was writing from my cellphone and I made a mistake. Let me rephrase it properly for clarity.

- If you use `units: ‘meters’` the object will always consider latitude, even if you move from its original position. Basically on every `obj.set()` or `obj.setCoords()` the scale units you use as value in `scale: ...` is multiplied by the size of a meter in that latitude.
- If you use `units: 'scene'` the object will ignore latitude, BUT the value you will need to use in `scale: ...`, so if you change the position it will always have the same size. is obviously different from the value you have been using with meters.
- You can check the [documentation](https://github.com/jscastro76/threebox/blob/master/docs/Threebox.md#loadobj)
- I’d recommend you to check the example [Mercator](https://github.com/jscastro76/threebox/blob/master/examples/04-mercator.html) and change the line 93 from `meters` to `scene`
- To your question, yes, init options for 3D Objects are cloned too when you use obj.duplicate()

**Comment by jscastro76** [comment]
@joedjc did my answer solved your question? May I close the issue?

**Comment by joedjc** [comment]
@jscastro76 I haven't tried it yet but it sounds like I will. I will close this issue. Many thanks for your help

**Comment by jscastro76** [comment]
> @jscastro76 I haven't tried it yet but it sounds like I will. I will close this issue. Many thanks for your help

@joedjc if not, let me know. Thanks for the feedback
